crates/bws/src/main.rs

```diff
@@ -120,6 +121,30 @@ async fn process_commands() -> Result<()> {
             command::secret::process_command(cmd, client, organization_id, output_settings).await
         }
 
+        Commands::Run {
+            command,
+            shell,
+            no_inherit_env,
+            project_id,
+            uuids_as_keynames,
+        } => {
+            match command::run::run(
+                client,
+                organization_id,
+                project_id,
+                uuids_as_keynames,
+                no_inherit_env,
+                shell,
+                command,
+            )
+            .await
+            {
+                // exit with the exit code from the child process
+                Ok(exit_code) => std::process::exit(exit_code),
+                Err(e) => Err(e),
+            }
```

Comment:
You can rewrite this as:

```suggestion
            let exit_code = command::run::run(
                client,
                organization_id,
                project_id,
                uuids_as_keynames,
                no_inherit_env,
                shell,
                command,
            )
            .await?;

            // exit with the exit code from the child process
            std::process::exit(exit_code);
```

which removes the match and is easier to follow.

Reply:
Thanks! Applied in e86aa2e06503900c31ade3e01649ad73c658ed82.